modules/ztp-configuring-hwevents-using-pgt.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-creating-hwevents_{context}"]
+= Configuring bare-metal events monitoring using PolicyGenTemplate CRs
```

Comment:
Maybe `events` -> `event` in the title here. I think it reads a little clearer, but if it changed the context you can ignore 🙂